.github/workflows/tests.yml

```diff
@@ -45,11 +45,14 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
           cache: pip
-          cache-dependency-path: setup.cfg
+          cache-dependency-path: |
+            setup.cfg
+            pyproject.toml
 
       - name: Install dependencies
         run: |
           pip install -e .[$EXTRAS]
+          pip install importlib_metadata
```

Comment:
Why do I need to explicitly install `importlib_metadata` if it's already defined in `pyproject.toml`?
This was not needed in `idutils`.